src/sentry/models/group.py

```diff
@@ -535,6 +535,10 @@ def get_event_metadata(self) -> Mapping[str, str]:
 
     @property
     def title(self) -> str:
+        current_tree_label = self.get_event_metadata().get("current_tree_label")
+        if current_tree_label is not None:
+            return " | ".join(current_tree_label)
```

Comment:
`" | ".join` appears in four places as far as I can see. Should we create a helper function for this somewhere?

Reply:
fair enough